NEWS.md

```diff
@@ -1,3 +1,6 @@
+## gDRcore 1.5.3 - 2024-12-18
+* fix melt error after changed `intersect` behaviour
```

Comment:
How has the `intersect` behavior changed? Should we inspect the use of `intersect` across the gdrplatform codebase?

Reply:
Now it drops names, so indexing object with names won't work after `intersect` use

Reply:
I've added a task to evaluate it further: https://jira.gene.com/jira/browse/GDR-2813.